### PR TITLE
Preparing release v1.38.0

### PR DESCRIPTION
**Pull request by rubenvp8510**
Signed-off-by: Ruben Vargas <ruben.vp8510@gmail.com>

## Which problem is this PR solving?
- Resolves https://github.com/jaegertracing/jaeger-operator/issues/2067

## Short description of the changes
-  Preparation for release 1.38.0, update manifests, changelog etc..

### CODE REVIEW

**Comment by frzifus** [comment]
https://github.com/jaegertracing/jaeger-operator/actions/runs/3079806755/jobs/4976453790#step:6:265

File: bundle/manifests/jaeger-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -23,11 +23,11 @@ metadata:
     description: Provides tracing, monitoring and troubleshooting for microservices-based
       distributed systems
     operators.openshift.io/infrastructure-features: '["disconnected"]'
-    operators.operatorframework.io/builder: operator-sdk-v1.23.0
+    operators.operatorframework.io/builder: operator-sdk-v1.22.0
```
Comment (frzifus) [inline bundle/manifests/jaeger-operator.clusterserviceversion.yaml:26]: i think just undoing this line should fix the build.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
 Changes by Version
 ==================
 
+1.38.0 (2022-09-19)
+-------------------
+* Set alias enable variable for spark cronjob ([#2061](https://github.com/jaegertracing/jaeger-operator/pull/2061), [@miyunari](https://github.com/miyunari))
```
Comment (frzifus) [inline CHANGELOG.md:6]: note i added https://github.com/jaegertracing/jaeger-operator/pull/2066 this morning.

**Review by frzifus — APPROVED** [review]

**Review by frzifus — DISMISSED** [review]

**Comment by rubenvp8510** [comment]
```
+---------------------------------------------------------+--------+
|                          NAME                           | RESULT |
+---------------------------------------------------------+--------+
| examples/examples_service_types                         | passed |
| examples/examples_with_sampling                         | passed |
| examples/examples_with_cassandra                        | passed |
| examples/examples_with_badger_and_volume                | passed |
| examples/examples_with_badger                           | passed |
| examples/examples_simplest                              | passed |
| examples/examples_simple_prod_with_volumes              | passed |
| examples/examples_simple_prod                           | passed |
| examples/examples_business_application_injected_sidecar | passed |
| examples/examples_openshift_agent_as_daemonset          | passed |
| examples/examples_collector_with_priority_class         | failed |
| examples/examples_openshift_with_htpasswd               | passed |
| examples/examples_agent_with_priority_class             | passed |
| examples/examples_auto_provision_kafka                  | passed |
| examples/examples_agent_as_daemonset                    | passed |
+---------------------------------------------------------+--------+


+----------------------------------------------+--------+
|                     NAME                     | RESULT |
+----------------------------------------------+--------+
| miscellaneous/collector_otlp_production_grpc | passed |
| miscellaneous/set_custom_img                 | passed |
| miscellaneous/collector_otlp_production_http | passed |
| miscellaneous/collector_otlp_allinone_grpc   | passed |
| miscellaneous/collector_otlp_allinone_http   | passed |
| miscellaneous/collector_autoscale            | passed |
+----------------------------------------------+--------+


+-----------------------------------------------------+--------+
|                        NAME                         | RESULT |
+-----------------------------------------------------+--------+
| streaming/streaming_with_autoprovisioning_autoscale | failed |
| streaming/streaming_with_tls                        | passed |
| streaming/streaming_simple                          | passed |
+-----------------------------------------------------+--------+

+------------------------------+--------+
|             NAME             | RESULT |
+------------------------------+--------+
| sidecar/sidecar_namespace    | passed |
| sidecar/sidecar_skip_webhook | passed |
| sidecar/sidecar_deployment   | passed |
+------------------------------+--------+


+---------------+--------+
|     NAME      | RESULT |
+---------------+--------+
| ui/allinone   | passed |
| ui/production | passed |
+---------------+--------+
```

Only two tests are failing @iblancasa can you take a look?

 examples/examples_collector_with_priority_class  is returning me this for the assert job:

```
time="2022-09-21T04:45:24Z" level=info msg="Secret provided for the Authorization header"
time="2022-09-21T04:45:24Z" level=info msg="Polling to https://collector-with-high-priority-query/api/services"
time="2022-09-21T04:45:32Z" level=info msg="Doing request number 0"
time="2022-09-21T04:45:32Z" level=warning msg="Status code: 403"
```

The other one seems to be a flaky test: streaming/streaming_with_autoprovisioning_autoscale

**Comment by iblancasa** [comment]
@rubenvp8510 thanks for the results.
What version of OpenShift are you using?

Also, `streaming/streaming_with_autoprovisioning_autoscale` is not flaky. You need to adjust the value of `E2E_TESTS_TIMEOUT` (described in [CONTRIBUTING.md](https://github.com/jaegertracing/jaeger-operator/blob/main/CONTRIBUTING.md#runing-the-e2e-tests)) to be increased depending on the resources of your OCP cluster.

**Comment by iblancasa** [comment]
I was able to run `examples/examples_collector_with_priority_class ` and pass it without any issues. I ran it 5 times with the same result.


The tests related to autoprovisioning should not be working on OpenShift due to the changes done in #2060.
I don't understand why  `examples/examples_auto_provision_kafka` was reported as `passed`.

I'll provide a PR with the fix soon.

**Review by frzifus — APPROVED** [review]

**Comment by iblancasa** [comment]
After doing some testing, I think we can merge.

There is something that needs to be fixed in the automation but I'll create a PR ASAP.

**Pull request merged**